### DISCUSSION TITLE
Motion event not started  on initial startup of Motionplus but does on a Restart (web

### DISCUSSION THREAD

**Discussion by martin762** [Motion-Project/motionplus#36] [Q&A] [answered]
To mister Dave an co developers many thanks for your Motionplus work.

I am using both Ubuntu20.4 on a larger computer desktop and a Raspberrypi 4(PiOS with both libcamera enabled and /dev/video)). My observation concerns both. When I enter the command motionplus, everything starts okay and i see the text "motion enabled" in the logs. However no motion is recorded from my cameras. I can see that at least with a webcam the light is on. Movement in front of any camera (pi module or webcam does not start an event. No file saved or log info.  Nothing. The cameras are working as I can see their output on localhost:8080.

When I issue a RESTART command from the web control (localhost:8080) I observe on the logs  that it restarts correctly and now the the video file is saved. The info is recorded in logs (as event start, event finish etc.) This applies to both Ubuntu and PiOS on the respective hardware.

Is this a correct action? Am I missing something in the config file?

For both use cases I have added a camera1.conf file and for the RaspberryPI a camera1.conf and Camera2.conf files, with mainly defaults but enabled the webcontrol. option3. The motionplus is compiled from the master (13 Feb) in both cases and the compile was successful after following build instructions. I stop with a controlC or kill command. The next start after either gives the same issue corrected by the remote RESTART. Is the  shutdown not working properly?

I have started a discussion rather than a new issue just in case I have missed something obvious. Many thanks for your consideration and advice.

PS: The libwebcam module is working well and I suggest that the warning in the script may be cast to history in the near future.

**Comment by Mr-Dave** [comment, marked as answer]
There was an error in the threshold_maximum and a fix has been pushed.

**Reply by martin762** [reply]
Many thanks for your response and the correction. I did try upping the threshold_maximum in the config, with no apparent change. However I see that you have changed the source code so I will recompile, and look forward to reporting a good result.

**Reply by martin762** [reply]
Hello Mr Dave
I can confirm that your change has corrected the above issue . I am seeing good results on both the desktop (ubuntu) and my Raspberrypi4 (with libcamera and webcam including both together).  I also observe that by using the prefix libcamerify and /dev/video0 I see a stable result.
One further observation. I only see the cameras working if I enable the camerax.conf files. With the "motionplus.conf" alone and a single camera (or module) the motionplus does not throw an error. but the camera is not activated (no light). Is this by design for motionplus?

Finally the camera module with libcam is a little fussy about the format and throws a wobbly if it does not like it e.g. default on your camera2.conf example. A libcam issue I am sure.

**Reply by martin762** [reply]
For the record my "one further observation ......" above was answered in Discussion  #41
